Assets/Modules/Enemies/ProjectileEntity.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ProjectileEntity : MonoBehaviour
+{
+
+    public float projectileSpeed = 7;
+    public Rigidbody2D rb;
+
+    private float timer = 0;
+    public Vector2 thisObjectsVelocity;
```

Comment:
One tip for naming variables, the name should not have information about the context, because we already have that information. For example here, `velocity` would be enough. because whoever tries to access this value will already know that this value exists in this `ProjectileEntity` instance

Reply:
Done